src/components/Confirm/Previews.js

```diff
@@ -1,6 +1,6 @@
 import { h } from 'preact'
 import classNames from 'classnames'
-import { localised } from '../../locales'
+import { useLocales } from '../../locales'
```

Comment:
I would say this is a good conversion, but seems a bit unrelated to the whole PR

Reply:
Ah, this is my bad. I was initially going to try to do a TS conversion but decided against it & left this behind. Will revert it to have only relevant changes in this PR.